public/electron.js

```diff
@@ -220,6 +230,8 @@ if (!gotTheLock) {
 
     ipcMain.handle('updater:quitAndInstallUpdate', () => autoUpdater.quitAndInstall())
 
+    ipcMain.handle('wc:hideApp', () => app.hide())
```

Comment:
I wonder if we should associate the `hideApp` action to wc. This might be used differently in the future. Would a `app` namespace make sense? `app:hide`. ?

Reply:
It would! But at the moment it's only part of the `wc` workflow, so I'd personally leave it there until the moment other workflows need to hide the app as well. WDYT?

Reply:
mhh hiding the app is such a generic action that it doesn't really make sense to me to associate it to `wc`.
What would be the drawbacks of using `app:hide`?

Reply:
no drawbacks at all, I'll make the change!

Reply:
done!